autogluon/task/image_classification/classifier.py

```diff
@@ -112,37 +114,83 @@ def predict(self, X, input_size=224, plot=True):
         >>> test_data = task.Dataset('~/data/test', train=False)
         >>> class_index, class_probability = classifier.predict('example.jpg')
         """
-        # model inference
+
         input_size = self.model.input_size if hasattr(self.model, 'input_size') else input_size
-        resize = int(math.ceil(input_size / 0.875))
-        transform_fn = Compose([
-                Resize(resize),
-                CenterCrop(input_size),
-                ToTensor(),
-                transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
-            ])
-        def predict_img(img):
+        resize = int(math.ceil(input_size / resize_ratio))
+
+        transform_size = transforms.Compose([
+            transforms.Resize(resize),
+            transforms.CenterCrop(input_size),
+            transforms.ToTensor(),
+            transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
```

Comment:
why are hardcoded?

Reply:
All the GluonCV model or even PyTorch pretrained models are using this normalization mean and std, as it is from ImageNet dataset. It is okay to hardcode, because we only do transfer learning for `ImageClassifcation.fit`